x-pack/test/functional/services/ml/data_frame_analytics_creation.ts

```diff
@@ -88,6 +88,14 @@ export function MachineLearningDataFrameAnalyticsCreationProvider(
       await testSubjects.existOrFail('mlDFAnalyticsJobCreationJobDescription');
     },
 
+    async assertJobDescriptionEditInputExists() {
```

Comment:
I think all the new service methods around job editing should have their own service file, because this one (`data_frame_analytics_creation`)  is used for the creation process, i.e. the wizard.

Reply:
Created the new service in 64abf306dfb6f2021e1536a2542b4631ba07de6a